tests/epyccel/test_epyccel_types.py

```diff
@@ -6,6 +6,9 @@
 from modules import types
 from pyccel.epyccel import epyccel
 
+RTOL = 2e-14
+ATOL = 1e-15
+

```

Comment:
All functions in the [**types**](https://github.com/pyccel/pyccel/blob/master/tests/epyccel/modules/types.py) module are simple identities, hence it is appropriate to use exact comparison here.